sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -623,9 +623,9 @@ private bool TryGetAttributeProperties(AttributeData attributeData, Location? at
 
                 foreach (var namedArgument in attributeData.NamedArguments)
                 {
-                    if (namedArgument.Value.Value != null)
+                    if (IsNamedArgumentValid(namedArgument.Value))
                     {
-                        if (string.Equals(namedArgument.Key, Constants.FunctionMetadataBindingProps.IsBatchedKey) && !attrProperties.ContainsKey("cardinality"))
+                        if (string.Equals(namedArgument.Key, Constants.FunctionMetadataBindingProps.IsBatchedKey) && !attrProperties.ContainsKey("cardinality") && namedArgument.Value.Value != null)
```

Comment:
nit: getting a bit long, can we break this down?
```suggestion
                        if (string.Equals(namedArgument.Key, Constants.FunctionMetadataBindingProps.IsBatchedKey)
                        && !attrProperties.ContainsKey("cardinality") && namedArgument.Value.Value != null)
```